analyzer/src/main/resources/scripts/init.gradle

```diff
@@ -251,7 +251,7 @@ class AbstractDependencyTreePlugin<T> implements Plugin<T> {
         /**
          * Return true if the provided DependencyResult represents an imported Maven BOM.
          */
-        private boolean isBom(DependencyResult dependencyResult) {
+        private static boolean isBom(DependencyResult dependencyResult) {
```

Comment:
Nit: Unrelated change.

Reply:
Ah, sorry, expanding the diffs shows that it's not unrelated as `isBom` is called from the static `dependencyTreeEquals` now.